src/ImageTransformations.jl

```diff
@@ -1,3 +1,40 @@
+"""
+
+This package provides support for image resizing, image rotation, and other spatial transformations of arrays.
+
+- `InvWarpedView`: Create a view of `img` that lazily transforms any given index `I` passed to `wv[I]` so that `wv[I] == img[inv(tinv)(I)]`.
+- `WarpedView`: Create a view of `img` that lazily transforms any given index `I` passed to `wv[I]` so that `wv[I] == img[tform(I)]`.
```

Comment:
Could you perhaps list `WarpedView` first since it's conceptually the simpler of the two?